### DISCUSSION TITLE
template help

### DISCUSSION THREAD

**Discussion by jhs1965net** [windingwind/zotero-better-notes#474] [Note Templates]
Hello , thanks for your great work ,  but it impossible for me that I don't know java to understand how to use templates and the help page doesn't work.
thanks a lot

James

**Comment by windingwind** [comment]
Hi,

Using template doesn't mean you have to know how to write JS. The doc is still on the way. I am a little bit busy these days.

If you want to use community templates provided here, you could roll back to 0.8.9 and they should work. The docs for former versions is here: https://github.com/windingwind/zotero-better-notes/wiki/4.2%20Note%20Template-Note%20Template.en

Please do not create duplicate threads, thanks.

**Reply by jhs1965net** [reply]
this an example

Template [Text] table Error: SyntaxError: "" string literal contains an unescaped line break
